src/Signature.ts

```diff
@@ -0,0 +1,50 @@
+import {
+    Key,
+    KeyType,
+    signatureToString,
+    stringToSignature,
+} from './eosjs-numeric';
+import BN = require('bn.js');
+
+/** Represents/stores a Signature and provides easy conversion for use with `elliptic` lib */
+export class Signature {
+
+    constructor(private signature: Key) {}
+
+    /** Instantiate Signature from an EOSIO-format Signature */
+    public static fromString(sig: string): Signature {
+        return new Signature(stringToSignature(sig));
+    }
+
+    /** Instantiate Signature from an `elliptic`-format Signature */
+    public static fromElliptic(ellipticSig: any): Signature {
```

Comment:
I think elliptic provides types info, so you can get rid of more of these `any`s.

https://www.npmjs.com/package/@types/elliptic

Reply:
They were borked last I tried. Maybe they've improved since then.

Reply:
I wish that scenario wasn't as common as it is.